packages/tokens/src/tokens/color/component-dark.json

```diff
@@ -1,7 +1,7 @@
 {
-  "button-primary-background-on-dark": {
-    "name": "button-primary-background-on-dark",
-    "value": "{semantic-test-on-dark}",
+  "vads-segmented-control-color-surface-selected-on-dark": {
+    "name": "vads-segmented-control-color-surface-selected-on-dark",
+    "value": "{vads-color-base-dark}",
```

Comment:
Due to the additional level of nesting that VADS has added to their tokens, we need to tweak references to them slightly.

When referencing any of the `vads-*` tokens created by the web team, we also need to append `.*.value`.  e.g.:

```
    "value": "{vads-color-base-dark.*.value}",
```

You can tell which ones need the additional `.*.value` by looking at [variables.json](https://github.com/department-of-veterans-affairs/component-library/blob/426188b6cc7af2c1f22b0e517e40e41e2d627f06/packages/css-library/dist/tokens/json/variables.json#L109) and seeing if there's a `*` level